src/api.raml

```diff
@@ -176,6 +176,14 @@ traits:
                   - published
                 required: false
                 default: default
+    contained:
```

Comment:
strange name `contained` since this item is the one that contains others (maybe `container`)